src/components/tag-input/tag-input.component.ts

```diff
@@ -192,6 +192,9 @@ export class TagInputComponent<T = any> implements AfterContentInit, OnChanges,
     /** Raised when a tag has been clicked. The `tag` property of the event contains the clicked tag. Call `preventDefault()` on the event to prevent the default behaviour of selecting the tag. */
     @Output() tagClick = new EventEmitter<TagInputEvent>();
 
+    // When clicking on the input during mutliple mode it will send a on touched event to the parent component
+    @Output() inputFocus = new EventEmitter<TagInputEvent>();

```

Comment:
Currently we are not emitting anything, so `TagInputEvent` would be incorrect, this would be `void`, however with my previous comment of emitting the native focus event we should have a type of `FocusEvent` I believe.

The docs would also need updated.

Reply:
Thinks I've updated this correctly, let me know if there is any further changes!